c2corg_api/emails/email_service.py

```diff
@@ -38,6 +38,13 @@ def send_registration_confirmation(self, lang, user, link):
                 subject='Inscription sur Camptocamp.org',
                 body='Pour activer votre compte cliquez sur %s' % link)
 
+    def send_request_change_password(self, lang, user, link):
+        # TODO: handle i18n
+        self._send_email(
+                user.email,
+                subject='Changement de mot de passe sur Camptocamp.org',
```

Comment:
I disagree. All interface strings are written in english in the code, why using french for emails? Especially if it is a temporary solution: it's not a problem for testers if they receive messages in english.